cli/tests/integrations/test_marathon_pods.py

```diff
@@ -0,0 +1,173 @@
+import contextlib
+import json
+import re
+
+from .common import exec_command, file_bytes, file_json
+
+FILE_PATH_BASE = 'tests/data/marathon/pods/'
+
+GOOD_POD_ID = 'good-pod'
+GOOD_POD_FILE_PATH = FILE_PATH_BASE + 'good.json'
+UPDATED_GOOD_POD_FILE_PATH = FILE_PATH_BASE + 'updated_good.json'
+
+DOUBLE_POD_ID = 'double-pod'
+DOUBLE_POD_FILE_PATH = FILE_PATH_BASE + 'double.json'
+
+TRIPLE_POD_ID = 'winston'
+TRIPLE_POD_FILE_PATH = FILE_PATH_BASE + 'doubleplusgood.json'
+
+
+def test_pod_add_from_file_then_remove():
+    exit_status = _pod_add_from_file(GOOD_POD_FILE_PATH)
+    assert exit_status == 0
+
+    exit_status = _pod_remove(GOOD_POD_ID, force=False)
+    assert exit_status == 0
+
+
+def test_pod_add_from_stdin_then_force_remove():
+    exit_status = _pod_add_from_stdin(GOOD_POD_FILE_PATH)
+    assert exit_status == 0
+
+    exit_status = _pod_remove(GOOD_POD_ID, force=True)
+    assert exit_status == 0
+
+
+def test_pod_list():
+    paths = [GOOD_POD_FILE_PATH, DOUBLE_POD_FILE_PATH, TRIPLE_POD_FILE_PATH]
+    expected_json = [file_json(path) for path in paths]
+
+    with _pod(GOOD_POD_ID, GOOD_POD_FILE_PATH), \
+            _pod(DOUBLE_POD_ID, DOUBLE_POD_FILE_PATH), \
+            _pod(TRIPLE_POD_ID, TRIPLE_POD_FILE_PATH):
+
+        exit_status, stdout = _pod_list(json=True)
+        assert exit_status == 0
+
+        parsed_stdout = json.loads(stdout.decode('utf-8'))
+        _assert_same_elements(parsed_stdout, expected_json)
```

Comment:
In this case, it's not straightforward in Python because the lists contain dicts. So I can't turn the lists into sets because dicts aren't hashable, and sorting is tricky because dict entries are unordered. I could (_recursively_!) convert the dicts into lists of key-value tuples sorted by key, but at that point `assert_same_elements` is simpler.

If you were wondering why I added this method in my previous PR, now you know :stuck_out_tongue_closed_eyes: